From 7576264dbc201d88220b056009d216b468bfe70d Mon Sep 17 00:00:00 2001
From: Fabrizio Castro <fabrizio.castro.jz@renesas.com>
Date: Thu, 16 Jun 2022 16:50:36 +0100
Subject: [PATCH] arm64: dts: rzg2l-smarc-som: Disable ethernet by default

Although the ethernet PHYs are located on the SMARC SoM, the connectors
are located on the carrier board, therefore disable them by default and
have the carrier board enable the interface(s) actually used.

Signed-off-by: Fabrizio Castro <fabrizio.castro.jz@renesas.com>
---
 arch/arm64/boot/dts/renesas/rzg2l-smarc-som.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/rzg2l-smarc-som.dtsi b/arch/arm64/boot/dts/renesas/rzg2l-smarc-som.dtsi
index 29fc1e4db2e3..200c70fdbaba 100644
--- a/arch/arm64/boot/dts/renesas/rzg2l-smarc-som.dtsi
+++ b/arch/arm64/boot/dts/renesas/rzg2l-smarc-som.dtsi
@@ -129,7 +129,7 @@ &eth0 {
 	pinctrl-names = "default";
 	phy-handle = <&phy0>;
 	phy-mode = "rgmii-id";
-	status = "okay";
+	status = "disabled";
 
 	phy0: ethernet-phy@7 {
 		compatible = "ethernet-phy-id0022.1640",
@@ -158,7 +158,7 @@ &eth1 {
 	pinctrl-names = "default";
 	phy-handle = <&phy1>;
 	phy-mode = "rgmii-id";
-	status = "okay";
+	status = "disabled";
 
 	phy1: ethernet-phy@7 {
 		compatible = "ethernet-phy-id0022.1640",
-- 
2.34.1

